### ISSUE TITLE
Adding folder to queue also add cover jpg

### ISSUE THREAD

**Issue by batric** [mkiol/Jupii#54]
![immagine](https://github.com/user-attachments/assets/f614e497-8d0c-4a1d-85d3-4890a49186aa)
Hello, adding folders works great, but also add cover.jpg (did not check for other file type actually).

The queue seems to be stuck playing the jpg

Many thanks
Riccardo

**Comment by mkiol** [comment]
This is not a bug but the expected behavior. You can add audio, video or image files to the play queue. For example, adding video or images makes sense for a TV. At the moment, Jupii is not able to detect the capabilities of the rendering device it is connected to, so all media types are allowed.

**Comment by batric** [comment]
that's correct, however I would expect to see a slideshow of pictures and not a still image; in this case putting a picture in the queue stops playing.

Maybe a timeout of 5/10 seconds before skipping to the next image/song could be handy.

Regards
Riccardo

**Comment by mkiol** [comment]
> however I would expect to see a slideshow of pictures and not a still image; in this case putting a picture in the queue stops playing. Maybe a timeout of 5/10 seconds before skipping to the next image/song could be handy.

This behavior (stopping playback) is caused by the renderer, not by Jupii :/

In my experience, "image playback" is broken on most devices (such as TVs). To overcome this problem, I plan to implement in Jupii an automatic conversion from image to video with a repeating single frame. The video is usually handled correctly on the TV. Video also has the nice property that you can set the duration and thus control how long it plays.

**Comment by mkiol** [comment]
It makes sense. Maybe as an additional option in the settings. Thanks for the idea.
